lib/infrastructure/core/dnd.dart

```diff
@@ -0,0 +1,18 @@
+import 'dart:io';
+
+import 'package:flutter/material.dart';
+import 'package:flutter_dnd/flutter_dnd.dart';
+
+Future<void> toggleDND(BuildContext contect) async {
```

Comment:
Please write toggleDnd

Reply:
Please delete `BuildContext contect` it is not in use

Reply:
This code should be encapsulated inside a class.
For an example of the structure please look at
PlayerController with _PlayerRepository 
or
VibrationController with _VibrationRepository

Reply:
I have made the changes as requested